compiler-core/templates/documentation_page.html

```diff
@@ -7,3 +7,9 @@
 {% block content %}
 {{ content|safe }}
 {% endblock %}
+
+{% block head -%}
+{% if !host.is_empty() && !project_name.is_empty() -%}
+<link rel="canonical" href="{{ host }}/{{ project_name|safe }}/{{ file_path|safe }}" />
+{%- endif %}
+{%- endblock %}
```

Comment:
Is there a reason not to have this in the layout? It seems to be the same for both templates.

Reply:
Originally was more direct to put it where I knew it was being used, but has converged enough that I was able to move it directly into the layout.